extensions/Khronos/KHR_draco_mesh_compression/README.md

```diff
@@ -0,0 +1,138 @@
+
+## Conformance
+
+To process this extension, there are some changes need to be made in loading a glTF asset.
+* If `KHR_draco_mesh_compression` is in `extensionsRequired` then the loader must support the extension or it will fail loading the assets.
+* If `KHR_draco_mesh_compression` is in `extensionsUsed` but not `extensionsRequired`:
+    * Check if the loader supports the extension. If not, then load the glTF asset ignoring the compression extension in `primitive`.
+    * If the loader supports the extension, then process the extension and ignore the attributes and indices of the primitive that are contained in the extension.
+* When encountering a `primitive` with the extension the first time, you must process the extension first. Get the data from the pointed `bufferView` in the extension and decompress the data to a geometry of a specific format, e.g. Draco geometry.
+* Then, process `attributes` and `indices` properties of the `primitive`. When loading each `accessor`, go to the previously decoded geometry in the `primitive` to get indices and attributes data. A loader could use the decompressed data to overwrite `accessors` or render the decompressed geometry directly (e.g. ThreeJS).
+
+It is pretty straightforward for top-down loading of a glTF asset, e.g. only
+decompress the geometry data when a `primitive` is met for the first time. However, for
+bottom-up loading, loading `accessor` before `primitive` will not get the data. It could only be handled when processing its parent `primitive`. This is based on the consideration that it will rarely happen that
+loading an `accessor` without knowing its parent `primitive`. And it should be
+easy enough to change the loader to ignore `accessor` without `bufferView` in glTF 2.0. But we are
+definitely open to change this if there actually are some use cases that require
+loading `accessor` independently. 
+
+The extension provides compressed alternatives to one or more of a primitive's uncompressed attributes. A loader may choose to use the uncompressed attributes instead — when the extension is not supported, or for other reasons. When using compressed Draco data, the corresponding uncompressed attributes defined by the primitive should be ignored. If additional uncompressed attributes are defined by the primitive, but are not provided by the Draco extension, the loader must proceed to use these additional attributes as usual.
+
+* Implementation note: To prevent transmission of redundant data, exporters should generally write compressed Draco data into a separate buffer from the uncompressed fallback, and shared data into a third buffer. Loaders may then optimize to request only the necessary buffers.
+
+If the uncompressed version of asset is not provided for the fallback described above, then the extension must be added to `extensionsRequired` so that the loaders/engines don't support the extension could report failure.
```

Comment:
I think we can remove this line with the re-worked paragraph above.

Reply:
Done